tests/integration/components/links-with-follower-test.js

```diff
@@ -81,12 +86,75 @@ describeComponent(
     });
 
     describe('clicking a link', function() {
-      it.skip('moves follower to the active link position')
-      it.skip('changes width of follower to width of active link');
+      let router;
+
+      beforeEach(function() {
+        this.register('router:main', Ember.Object.extend(Ember.Evented));
+
+        router = this.container.lookup('router:main');
+
+        this.render(hbs`
+          {{#links-with-follower class="icon-nav"}}
+            <li class="active">One</li>
+            <li>Two</li>
+            <li class="long">Three Four Five Six</li>
+          {{/links-with-follower}}
+        `);
+      });
+
+      it('moves follower to the active link position', function(done) {
+        let $follower = this.$('.link-follower');
+
+        run.later(function() {
+          expect($follower.position().left).to.equal(this.$('.active').position().left);
+        }, 10);
+
+        run.later(function() {
+          this.$('.active').removeClass('active');
+          this.$('.long').addClass('active');
+          router.trigger('willTransition');
+        }, 20);
+
+        run.later(function() {
+          expect($follower.position().left).to.equal(this.$('.active').position().left);
+          done();
+        }, 30);
```

Comment:
Alright so I experimented with the two options above. Came up with the following:

- Using wait was nicer than using run.later, but I still ended up with some nesting that made things look a bit unclear.
- Using the explicit run loop doesn't wait until after the first render to run so the tests ended up failing
- I also experimented with using a nested run.scheduleOnce to hook into the current context and run the test once `afterRender` executed, but no dice there either.

What I did end up with was 2f3c646 which essentially wraps the expects in a singe run.later, set to 10ms, and then using an explicit run loop for simulating the `willTransition` hook on a router. I think this is a good middle ground for now as a test to ensure that the position and width are calculated correctly.